cktap/bip32.py

```diff
@@ -0,0 +1,591 @@
+import hmac
```

Comment:
copyright?

Reply:
no copyright needed here, it is part of my repo https://github.com/scgbckbone/btc-hd-wallet/

I did just based on the BIP specification long time ago...

Reply:
- sounds like you are the copyright holder then, if you implemented from BIP
- safe to put Coinkite's standard header block here then